.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Please update this template with something that matches your PR -->
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | no
```

Comment:
We should add a comment instead of this row. With a link to https://symfony.com/doc/current/contributing/code/bc.html maybe (and explaining we follow the same principle)?

Reply:
:warning: it'll be displayed for repositories JS, website... :(

Reply:
:warning: it'll be displayed for repositories JS, website... :(

Reply:
@toofff I suggest we use separate templates for each repository.

Reply:
Trying to have a common set of templates will backfire. It'll also make overriding them a nightmare.

Reply:
For PRs you mean? The local one override the global one. I think we can put a generic file here (we have many depo without templates at all), and create specific one for main repos such as core and admin.

Reply:
What about the issue templates? Will having any local template in a particular repository override all global templates?

Reply:
AFAIR yes. To be tested.